### ISSUE TITLE
JBatch poms on maven central refer to missing javax.batch pom

### ISSUE THREAD

**Issue by smillidge** [WASdev/standards.jsr352.jbatch#33]
Currently the JBatch container and spi 1.0.1-b03 poms on maven central refer to a javax.jbatch.batch pom of the same version 1.0.1-b03. However maven central only has 1.0.1-b01 installed.

**Comment by scottkurz** [comment]
Ah.. that's the parent POM.. which I neglected to update in not wanting to update the javax.batch-api POM.   

Thanks for pointing this out.  This actually may be part of what's slowed down the 1.0.1 integration testing.

**Comment by scottkurz** [comment]
I was actually thinking of moving the parent POM to the 'com.ibm.jbatch' group since I think the lifecycle will align most with the implementation rather than the API or TCK (In a separate change I'm moving the TCK to 'com.ibm.jbatch.tck' group).

I don't expect there are a lot of dependencies on our parent POM (javax.batch:jbatch).. only maybe the API (javax.batch:javax.batch-api).

Sound reasonable?   @BrentDouglas any thoughts?

**Comment by BrentDouglas** [comment]
I'm not sure. There are really three separate projects in this repo (the api, the TCK and the RI). If it were up to me I would structure it more like the CDI project (https://github.com/cdi-spec) with the api (and spec sources...) in one repo using the javax.batch groupid, and separate repos for the TCK and RI using the 'com.ibm.jbatch' groupId.

In any case I think it makes more sense for the spec API to be standalone and would consider copying any relevant config from the parent into the batch-api module's pom and removing its parent declaration. You could then rename the parent module's groupId to com.ibm.jbatch and have both the TCK and RI inherit from it so then the spec is all 'javax.batch' and the RI and TCK are all 'com.ibm.jbatch'.

**Comment by scottkurz** [comment]
Well.. I'd like to keep this in one repo for the time being... though I did go and separate this out into three distinct Maven groupId(s)... for the API, IMPL, and TCK.

(I also got rid of the tabs (in the POMs)... accidentally merged it in but now too lazy to unmerge that change.)

The 'com.ibm.jbatch' group should show up in Maven central in the next day at the '1.0.1-b04' level.

Thanks for helping with this issue.
